### DISCUSSION TITLE
How to find WinUI 3 controls demo code

### DISCUSSION THREAD

**Discussion by SZLMCL** [microsoft/WinUI-Gallery#861] [Q&A] [answered]
Hello,

I downloaded a Windows Store app with name: "WinUI 3 Controls Gallery". I search for the source code of this app. The links in the app navigates to this github site, but after I download the code and open it an UWP app will be opened with name: XamlControlsGallery (Universal Windows). Who can I find the WinUI 3 version of this solution? 

I would like integrate Synfusion WinUI controls for testing to this demo app, but it shows the following error:
"NU1202	Package Syncfusion.Grid.WinUI 19.4.0.48 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm. Package Syncfusion.Grid.WinUI 19.4.0.48 supports: net5.0-windows10.0 (.NETCoreApp,Version=v5.0)"

When I install Syncfusion UWP controls, it works without any error. 

Can you help me please how can I find this WinUI 3 demo source code?

Sorry if I ask something trivial, I getting started with WinUI a few days ago only. 

Thank you!

**Comment by SZLMCL** [comment, marked as answer]
Oh I already found it in the winui brach. Thank you!

**Reply by gregwoo-microsoft** [reply]
Perfect! Please keep in mind though that the code in the WinUI 3 branch is not up to date (the code for the Store version is not currently on there). We should update the WinUI 3 branch in the upcoming days and I'll make an announcement once it is there 😄

**Reply by SZLMCL** [reply]
Thank you, I look forward to it! :)

**Reply by SZLMCL** [reply]
Can you please replace the Microsoft.ProjectReunion nuget to Microsoft.WindowsAppSDK 1.0? The reunion nuget causes lot of errors when we install 3rd party controls to the project (like Syncfusion WinUI). After replace the nugets all working well. Thank you!

**Reply by marcelwgn** [reply]
Pull request #858 will update the WinUI 3 version to use the WinAppSDK 1.0.

**Reply by FinnurHrafn** [reply]
Looking forward to see the up to date source :)

**Reply by JonPerkinsOldGit** [reply]
> Perfect! Please keep in mind though that the code in the WinUI 3 branch is not up to date (the code for the Store version is not currently on there). We should update the WinUI 3 branch in the upcoming days and I'll make an announcement once it is there 😄

So it's now the middle of April and I still can't find it after searching for over half an hour. Good example code is so vital because there is a lot of old UWP code in the that is no longer valid which pollutes the reference material. When?

**Reply by gregwoo-microsoft** [reply]
I really appreciate your patience with this. The source code can be found in the winui3 branch. It has some additional changes that are not in the Store version so we are planning to update the Store app within the next few weeks.
